README.md

```diff
@@ -1,8 +1,14 @@
 # Google Student Training in Engineering Program
+This repo contains Sean Tisdale's portfolio and STEP projects. 
+## 3 favorites
```

Comment:
Were you planning to add something under "3 favorites"?

Reply:
I was originally but now I plan on putting it in my portfolio somewhere

Reply:
I was originally but I decided to move that to the portfolio. Also this was on a different branch then from the one we discussed today. Is there a way to just forget about it and I put the contents in my newer branch?

Reply:
Looks like you figured it out since you updated the other pull request! :)

Yeah, you can close the pull request and delete the branch

git branch has a subcommand for deleting the branch (can't remember exact syntax, you can look it up with git help branch)